openpgp/packet/config.go

```diff
@@ -156,6 +159,13 @@ func (c *Config) PublicKeyAlgorithm() PublicKeyAlgorithm {
 	return c.Algorithm
 }
 
+func (c *Config) CurveType() Curve {
```

Comment:
`CurveName`? (Here since it's in the config I guess it's obvious enough that we mean the generic one)

Reply:
OK